src/Common/src/Interop/Windows/Interop.Libraries.cs

```diff
@@ -32,5 +32,6 @@ internal static partial class Libraries
         internal const string Ws2_32 = "ws2_32.dll";
         internal const string Wtsapi32 = "wtsapi32.dll";
         internal const string CompressionNative = "clrcompression.dll";
+        internal const string CORE_WINRT = "api-ms-win-core-winrt-l1-1-0.dll";
```

Comment:
This casing is different from everything else in this file. What about `CoreWinRT` ?

Reply:
Renamed. The CORE_WINRT was copied from CoreRT repro..